### ISSUE TITLE
How to install on Windows?

### ISSUE THREAD

**Issue by TomFCarrion** [danielcardeenas/whatsapp-framework#224]
I keep getting the following error when trying to install
I tried removing sudo from the code and changing python3 for python but I can't install

Python version: 3.5.0
Windows 10 x64
MIGW64 
any thoughts? 😃 

./setup.sh
Installing yowsup libraries
--------------------------
zip_safe flag not set; analyzing archive contents...
C:\Users\tomfc\AppData\Local\Programs\Python\Python35\lib\site-packages\setuptools\dist.py:285: UserWarning: Normalizing '0.4.1-2' to '0.4.1.post2'
  normalized_version,
error: Setup script exited with error: [WinError 2] El sistema no puede encontrar el archivo especificado
zip_safe flag not set; analyzing archive contents...
yowsup.common.__pycache__.tools.cpython-35: module references __file__
yowsup.yowsup.common.__pycache__.tools.cpython-35: module references __file__
C:\Users\tomfc\AppData\Local\Programs\Python\Python35\lib\site-packages\setuptools\dist.py:285: UserWarning: Normalizing '0.4.1-2' to '0.4.1.post2'
  normalized_version,
error: Setup script exited with error: [WinError 2] El sistema no puede encontrar el archivo especificado
--------------------------
Configuring modules
--------------------------
[hihelp] All good...
[pokedex] Installing dependencies...
Command "c:\users\tomfc\appdata\local\programs\python\python35\python.exe -c "import setuptools, tokenize;__file__='C:\\Users\\tomfc\\AppData\\Local\\Temp\\pip-build-bq1chey4\\simplejson\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\tomfc\AppData\Local\Temp\pip-cg6xjxy3-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\tomfc\AppData\Local\Temp\pip-build-bq1chey4\simplejson
Error:[1]. Try running it with root privilages

**Comment by kidstell** [comment]
> Did you try running it with administrator?

yes and it changes nothing. the issue persist. the link here says its an issue with setuptools https://github.com/pypa/setuptools/issues/491

i would feed back if i find anything worthwhile. help if you can
